core-concepts/actions.md

```diff
@@ -110,17 +107,17 @@ export default {
 }
 ```
 
-## Usage with the Options API
+## Uso con la API de Opciones {#usage-with-the-options-api}
 
 <VueSchoolLink
   href="https://vueschool.io/lessons/access-pinia-actions-in-the-options-api"
   title="Access Pinia Getters via the Options API"
 />
 
-For the following examples, you can assume the following store was created:
+Para los próximos ejemplos supón que el siguiente almacén fue creado:
```

Comment:
I think that we can improve to: "Para los siguientes ejemplos, puedes asumir que se ha creado el siguiente almacén:". Do you agree?